src/app/handlers/command.handler.ts

```diff
@@ -15,45 +21,66 @@ export class CommandHandler implements IHandler {
     }
 
     const plugin = plugins[aliases[command.name]];
-
     const isDM = !message.guild;
 
     if (plugin) {
-      if ((isDM && !plugin.usableInDM) || (!isDM && !plugin.usableInGuild)) {
-        return;
-      }
+      await this._attemptRunPlugin(message, plugin, command, isDM);
+      return;
+    }
+
+    await this._tryFuzzySearch(message, command, isDM);
+  }
+
+  private async _tryFuzzySearch(message: types.IMessage, command: types.ICommand, isDM: boolean) {
+    const plugins = this.container.pluginService.plugins;
+    const aliases = this.container.pluginService.aliases;
+
+    const [mostLikelyCommand] = PLUGIN_NAMES.sort(
+      (a: string, b: string) => levenshtein(command.name, a) - levenshtein(command.name, b)
+    );
+
+    const embed = new MessageEmbed();
+    embed.setTitle('Command not found');
+    embed.setDescription(
+      `Did you mean \`!` +
+        `${mostLikelyCommand}${command.args.length ? ' ' : ''}${command.args.join(' ')}\`?`
+    );
+
+    const msg = await message.channel.send(embed);
+    await msg.react(this._CHECK_EMOTE);
+    await msg.react(this._CANCEL_EMOTE);
+
+    const collector = msg.createReactionCollector(
+      (reaction: MessageReaction, user: User) =>
+        [this._CHECK_EMOTE, this._CANCEL_EMOTE].includes(reaction.emoji.name) &&
+        user.id !== msg.author.id, //Only run if its not the bot putting reacts
```

Comment:
`//<space>`

Reply:
Bruh i swear i've looked for this setting in prettier and cant find it 😂